packages/bits/src/lib/expander/expander.component.html

```diff
@@ -24,6 +25,7 @@
         </div>
     </div>
     <div class="nui-expander__body"
+         role="list"
```

Comment:
`list` may be two specific since the content inside the expander isn't necessarily a list of things.

Reply:
will change to section